G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/ExcelActivateSheet/ShouldActivateSheet_WhenExcekActivateSheetCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,11 @@
+﻿excel.open
+
+excel.addsheet name ‴TestSheet1‴
+excel.setvalue value 1 row 1 colindex 1
+excel.addsheet name ‴TestSheet2‴
+excel.setvalue value 0 row 1 colindex 1
+excel.activatesheet name ‴TestSheet1‴
+excel.getvalue row 1 colindex 1 result ♥cellValue
+test ⊂♥cellValue⊃
+
+excel.close
```

Comment:
Please use test after excel.close. Otherwise failed tests leave instance of excel open